src/components/content/Login.jsx

```diff
@@ -69,6 +69,11 @@ const Login = () => {
           </form>
         </div>
       )}
+      {isLoading === true && (
+        <div className="text-center">
+          <img src="https://assets-v2.lottiefiles.com/a/4b96cd0e-1164-11ee-b18f-5fe294a16463/a7YAZLDNWQ.gif" className="w-1/2 h-auto mx-auto" alt="Centered Image" />
+        </div>
+      )}
     </div>
   )
 }
```

Comment:
This is the loading gif, and we needed to use multiple times in pages. so to implement DRY rule, kindly make it a component . then import and render it where we needed. This will help to do not recode each time.